### PR TITLE
added login and signup pages

### PR DESCRIPTION
**Pull request by vivek-guda**
Added Login and Signup pages
[Website preview](https://fashion-store-ecom.netlify.app/)

ecom-login-page
ecom-signup-page

### CODE REVIEW

File: src/pages/Login/Login.js [hunk 1]
```diff
@@ -2,7 +2,27 @@ import { Link } from "react-router-dom";
...
+        <input
+          className="login-input"
+          type="password"
+          placeholder="Enter password"
+        />
+        <Link className="login-btn" to="/">
+          Fake Login
+        </Link>
+        <p className="alternate-msg">
+          Don't have an account? <Link to="/signup">Sign up</Link>
```
Comment (Mohasin-Haque) [inline src/pages/Login/Login.js:21]: How about making a component?. But it is good you did like this.

**Review by Mohasin-Haque — COMMENTED** [review]
Your code is clean and good. You can merge it.

File: src/pages/Login/Login.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  height: 75vh;
+}
+
+.login-container {
+  width: 30rem;
+  padding: 3rem 2.5rem;
+  margin: 10rem auto;
+  display: flex;
+  flex-direction: column;
+  background-color: #e0e0e0;
```
Comment (srv-pranjal) [inline src/pages/Login/Login.css:12]: You can use variables for colors

File: src/pages/Login/Login.js [hunk 3]
```diff
@@ -2,7 +2,27 @@ import { Link } from "react-router-dom";
 import "./Login.css";
 
 const Login = () => {
-  return <div>Login</div>;
+  return (
+    <main className="login-main">
+      <div className="login-container">
+        <h1 className="login-title">Login</h1>
+        <label className="login-label">Email</label>
+        <input className="login-input" type="text" placeholder="Enter email" />
```
Comment (srv-pranjal) [inline src/pages/Login/Login.js:10]: It's a good practice to use form whenever we have input fields

**Review by srv-pranjal — APPROVED** [review]
Good Folder Structure.
Code is Clean.
Left some suggestions, rest everything looks good.

**Pull request merged**